### PR TITLE
USHIFT-492: Update, clarify getting_started doc

### PR DESCRIPTION
**Pull request by sallyom**
This is a minor PR to update the wording of the getting_started doc. It does not introduce changes to MicroShift function. 

/cc @ggiguash

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -51,25 +47,46 @@ virt-install \
...
+exit
 ```
 
-First, copy your pull secret file to the MicroShift virtual machine using `redhat:redhat` credentials.
+Copy your pull secret file to the MicroShift virtual machine using `redhat:redhat` credentials.
 
 ```bash
 USHIFT_IP=192.168.122.2
-scp ~/.pull-secret.json redhat@${USHIFT_IP}:
+sudo scp ~/.pull-secret.json redhat@${USHIFT_IP}:
```
Comment (ggiguash) [inline docs/getting_started.md:78]: Why is `sudo` required here? I do not think it is.

File: docs/getting_started.md [hunk 2]
```diff
@@ -51,25 +47,46 @@ virt-install \
 ```
 
 Watch the OS console of the virtual machine to see the progress of the installation, waiting until the machine is rebooted and the login prompt appears.
+The OS console is also accessible from the `virt-manager GUI` as a result of running `sudo virt-manager`.
```
Comment (ggiguash) [inline docs/getting_started.md:50]:
```suggestion
> Run the `sudo virt-manager` command to access the OS console in the `virt-manager` GUI.
```

File: docs/getting_started.md [hunk 3]
```diff
@@ -51,25 +47,46 @@ virt-install \
 ```
 
 Watch the OS console of the virtual machine to see the progress of the installation, waiting until the machine is rebooted and the login prompt appears.
+The OS console is also accessible from the `virt-manager GUI` as a result of running `sudo virt-manager`. 
 
 ## Access MicroShift
 
-Log into the hypervisor machine using your user credentials.
+From the OS console of the virtual machine, it is possible to log into the machine using your user credentials, `redhat:redhat`.
```
Comment (ggiguash) [inline docs/getting_started.md:54]:
```suggestion
It is possible to log into the virtual machine from the OS console or using SSH.
```

File: docs/getting_started.md [hunk 4]
```diff
@@ -51,25 +47,46 @@ virt-install \
...
 
-Is it most convenient to access the MicroShift virtual machine using SSH. Run the following command to get the machine IP address and use it to remotely connect to the system.
+However, it is most convenient to access the MicroShift virtual machine using SSH.
```
Comment (ggiguash) [inline docs/getting_started.md:56]:
```suggestion
```

File: docs/getting_started.md [hunk 5]
```diff
@@ -51,25 +47,46 @@ virt-install \
...
+ Name       MAC address          Protocol     Address
+-------------------------------------------------------------------------------
+ vnet2      52:54:00:6d:08:f7    ipv4         192.168.122.2/24
+```
+
+Then, verify the virtual machine is accessable using SSH and the user credentials, `redhat:redhat`.
+
+```bash
+sudo ssh redhat@192.168.122.2 # when prompted, password is `redhat`.
+exit
```
Comment (ggiguash) [inline docs/getting_started.md:71]:
```suggestion
```

This is not necessary

File: docs/getting_started.md [hunk 6]
```diff
@@ -51,25 +47,46 @@ virt-install \
...
-Log into the MicroShift virtual machine using `redhat:redhat` credentials. Run the following commands to configure `CRI-O` for using the pull secret and start the MicroShift service.
+Log into the MicroShift virtual machine.
+
+```bash
+sudo ssh redhat@192.168.122.2 # when prompted, password is `redhat`
+```
+
+The remaining commands are to be executed from within the virtual machine as the `redhat` user.
+
+Configure `CRI-O` to use the pull secret and start the MicroShift service.
```
Comment (ggiguash) [inline docs/getting_started.md:89]:
```suggestion
Log into the MicroShift virtual machine.
```bash
sudo ssh redhat@${USHIFT_IP}
```

The following commands are to be executed from within the virtual machine as the `redhat` user.

Configure `CRI-O` to use the pull secret and start the MicroShift service.
```

File: docs/getting_started.md [hunk 7]
```diff
@@ -51,25 +47,46 @@ virt-install \
...
 USHIFT_IP=192.168.122.2
-scp ~/.pull-secret.json redhat@${USHIFT_IP}:
+sudo scp ~/.pull-secret.json redhat@${USHIFT_IP}:
 ```
 
-Log into the MicroShift virtual machine using `redhat:redhat` credentials. Run the following commands to configure `CRI-O` for using the pull secret and start the MicroShift service.
+Log into the MicroShift virtual machine.
+
+```bash
+sudo ssh redhat@192.168.122.2 # when prompted, password is `redhat`
```
Comment (ggiguash) [inline docs/getting_started.md:84]:
```suggestion
ssh redhat@${USHIFT_IP} # when prompted, password is `redhat`
```

File: docs/getting_started.md (hunk 2 above)
Comment (ggiguash) [inline docs/getting_started.md:50]:
```suggestion
> Run the `sudo virt-manager` command to access the OS console in the `virt-manager` application user interface.
```

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: docs/getting_started.md (hunk 5 above)
Comment (sallyom) [reply docs/getting_started.md]: removed & pushed from local, thanks

File: docs/getting_started.md (hunk 7 above)
Comment (sallyom) [reply docs/getting_started.md:77]: turns out I had too many auth failures w/ non-root ssh bc ssh was trying all my old rsa keys I had in there, once I cleared them out I get the password prompt :)

Comment (sallyom) [reply docs/getting_started.md:77]: updated to remove sudo & pushed from local, thanks

File: docs/getting_started.md (hunk 1 above)
Comment (sallyom) [reply docs/getting_started.md]: turns out I had too many auth failures w/ non-root ssh bc ssh was trying all my old rsa keys I had in there, once I cleared them out I get the password prompt :) I was also very confused as to why that was necessary - now I know (it's not)

Comment (sallyom) [reply docs/getting_started.md]: removed the sudo & pushed the update - thanks!

**Comment by sallyom** [comment]
thanks @ggiguash for your review, i've straightened out the unnecessary sudo(s) and TIL there are limits to how many bad keys ssh will try b4 giving up

**Pull request merged**